lua/ibl/init.lua

```diff
@@ -187,10 +187,15 @@ M.refresh = function(bufnr)
     end
 
     local scope
+    local scope_start_line, scope_end_line
     if not scope_disabled and config.scope.enabled then
         scope = scp.get(bufnr, config)
         if scope and scope:start() >= 0 then
-            offset = top_offset - math.min(top_offset - math.min(offset, scope:start()), config.viewport_buffer.max)
+            local scope_start = scope:start()
+            local scope_end = scope:end_()
+            offset = top_offset - math.min(top_offset - math.min(offset, scope_start), config.viewport_buffer.max)
```

Comment:
The only reason I adjust offset here at all was to compensate for the scope calculation. This is no longer needed with this PR. You can just remove this line.

And maybe add a note in the docs that `viewport_buffer.max` is deprecated. But don't remove it, we don't want to make this a breaking change.

Reply:
I made a small update to the docs now. I think it would also make sense to add small change to the `min` docs too, noting that it is now the exact offset and not minimal. Should I just add that too?